.gitignore

```diff
@@ -34,3 +34,6 @@ dist-ssr
 
 # Crowdin
 env.d/development/crowdin
+
+# Typescript
+*.tsbuildinfo
```

Comment:
```suggestion
*.tsbuildinfo

```

Reply:
👍

Reply:
I also add a folder related to rollup. I'll update commit message during squash.